sdk-android/build.gradle.kts

```diff
@@ -54,6 +54,13 @@ android {
     buildTypes {
         getByName("debug") {
             setMatchingFallbacks("debug", "release")
+
```

Comment:
I don't get why this needs to be set as it is already in the default config section